### PR TITLE
Improve usage of neo-python CLI

### PR DESCRIPTION
**Pull request by jseagrave21**
**What current issue(s) does this address, or what feature is it adding?**
- Many actions receive user confirmation via prompt input (e.g. <password>, "confirm", etc.) but most do not handle a `KeyboardInterrupt` without crashing due to the unhandled error.
- Also, an option for password prompting for `ClaimGas` and `SplitUnspentCoin` still exists which is not consistent with the rest of neo-python
- Additionally, the description for "path" is wrong for `sc deploy`.
- Finally, a "fakepath" file remains whenever `make test` is run.

Supports https://github.com/CityOfZion/neo-python/issues/890 and I do not think it conflicts with https://github.com/CityOfZion/neo-python/issues/891 because the `KeyboardInterrupt` handling is integral to the user experience.

**How did you solve this problem?**
- add support for cancelling actions with `KeyboardInterrupt`
- remove the ability to circumvent a password prompt for `ClaimGas` and `SplitUnspentCoin`
- correct help documentation for `sc deploy` "path"
- cleanup "fakepath" file created in test_wallet.py

**How did you make sure your solution works?**
added tests and ran `make test`

**Are there any special changes in the code that we should be aware of?**
No

**Please check the following, if applicable:**

- [x] Did you add any tests?
- [x] Did you run `make lint`?
- [x] Did you run `make test`?
- [x] Are you making a PR to a feature branch or development rather than master?
- [x] Did you add an entry to `CHANGELOG.rst`? (if not, please do)

### CODE REVIEW

File: neo/Prompt/Commands/SC.py [hunk 1]
```diff
@@ -313,7 +325,7 @@ def execute(self, arguments):
             return False
 
     def command_desc(self):
-        p1 = ParameterDesc('path', 'path to the desired Python (.py) file')
+        p1 = ParameterDesc('path', 'path to the desired smart contract (.avm) file')
```
Comment (ixje) [inline neo/Prompt/Commands/SC.py]: 👍

**Comment by ixje** [comment]
Nice! 💪  
It would not be the first time that I accidentally cause some errors because I try to abort via a  keyboard by instinct.

**Comment by ixje** [comment]
200 points jseagrave

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/d5ae4a22ffe49912a2a3dc698d39ac63a8fcf2cef63ae136850d039c312591d8

**Pull request merged**